src/containers/Transaction/Transaction.test.js

```diff
@@ -67,6 +67,10 @@ const sampleData = [
 
 const history = createMemoryHistory();
 
+const amountFieldLabelName = capitalize(fields.amount);
+const priceFieldLabelName = capitalize(fields.price);
+const totalFieldLabelName = capitalize(fields.total);
```

Comment:
Would be nice to have that calculate functions/format functions moved somewhere. Maybe not now but the fact is that will need them to expose similar preview in eg. transaction history view.

Reply:
In case we don't store them in desired format already 🤔